Makefile

```diff
@@ -40,6 +44,18 @@ else
 		PACKAGE_VERSION := $(PRODUCT_VERSION)-$(BUILD_NUMBER)
 		ARCH_REPO_DIR := linux
 	endif
+	ifeq ($(UNAME_S),FreeBSD)
+		PLATFORM := linux

```

Comment:
Why not PLATFORM := freebsd ?

Reply:
it would be great to use "PLATFORM := freebsd" but all the compilation files would need to be changed without any improvments on the results. Since it changes nothing i prefered keep it like that. It needs few changes and everything is fine!

Reply:
which files? It seems to only be used in the Makefile and seems useless.

Reply:
you're right: in particular in Makefiles and QMake files too.
For the moment I prefer minimizing updates. When Makefiles will be removed we  (you?) will be able to do more proper things.